wp-includes/service-workers.php

```diff
@@ -86,7 +86,6 @@ function wp_register_service_worker_caching_route( $route, $args = array() ) {
  * @return string Service Worker URL.
  */
 function wp_get_service_worker_url( $scope = WP_Service_Workers::SCOPE_FRONT ) {
-	/* @var WP_Rewrite $wp_rewrite */
```

Comment:
Unrelated change?

Reply:
I was getting phpcs complaints about it, and since it wasn't needed, I just decided to remove it.

Reply:
Maybe worth adding a `@global WP_Rewrite $wp_rewrite WordPress rewrite rules object.` note to the doc-block instead, like common in core?

Reply:
Of course! 👍

Reply:
Done in 8749128916c9807474f8bd6671ff654ff6dafc3f